lib/widgets/list_of_cards.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:hearthstoneapp/src/model/hearthstone_card.dart';
+import 'package:hearthstoneapp/src/repository/card_list_repository.dart';
+import 'package:hearthstoneapp/utils/constants.dart';
+import 'package:hearthstoneapp/utils/custom_flex.dart';
+import 'package:hearthstoneapp/utils/palette.dart';
+import 'package:hearthstoneapp/widgets/card.dart';
+
+class ListOfCards extends StatelessWidget {
+  ListOfCards({Key? key}) : super(key: key);
+  final CardListRepository repo = CardListRepository();
```

Comment:
remember that it's not necessary to shorten names, rename "repo" pliz

Reply:
Done!